README.md

```diff
@@ -23,19 +23,17 @@ ONNX Runtime continuously strives to provide top performance for a broad and gro
 ### Alignment with ONNX Releases
 ONNX Runtime provides comprehensive support of the ONNX spec and can be used to run all models based on ONNX v1.2.1 and higher. See ONNX version release details [here](https://github.com/onnx/onnx/releases).
 
-As of March 2019, ONNX Runtime supports ONNX 1.4.
+As of May 2019, ONNX Runtime supports ONNX 1.5.
```

Comment:
We maintain a detailed version table here that maps onnxruntime/onnx/ir/opset/onnx-ml version compatibilities: https://github.com/Microsoft/onnxruntime/blob/master/docs/Versioning.md#version-matrix. I would also add a link to this page here.

Reply:
"ONNX opset version 
implemented in this release", for each onnxruntime version, this value should be a list. For example, the next ort release will support opset 7,8,9,10.

Reply:
This clarification is done in [this](https://github.com/Microsoft/onnxruntime/blob/master/docs/Versioning.md#backwards-compatibility) section. The idea is to say that in this release we've implemented a certain opset version and as per the backwards compatibility feature, we support previous versions as well.